Add dissable sandbox to the environment when openshot launches.

JacksonRG opened the pull request:
PR #4787 fixed this issue in our AppImages, but certain other setups were still encountering the issue of the timeline never showing up.

This PR moves this definition to `app.py`, so that it can apply to all ways of running openshot.

Tested the appimage, and it still shows timeline.

Commit:
Add dissable sandbox to the environment when openshot launches.

JacksonRG commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Dissable Sandbox when building from source](https://trello.com/c/co7s5EcL/148-dissable-sandbox-when-building-from-source)

jonoomph commented:
A few comments/questions:
* Does this need to be set before the `imports` of QtWebEngine*?
* Can we add some additional documentation to this (and some spacing), to give future devs an understanding of why this line of code exists? (example comment below)
* Should we remove our fix from https://github.com/OpenShot/openshot-qt/pull/4787/files, since including the ENV VAR in our `launch.py` script should work for all our builds - of course, this needs to be tested and verified though. But seems like 1 place in the code would be better than 2 places.

```
# Disable sandbox support for QtWebEngine (required on some Linux distros
# for the QtWebEngineWidgets to be rendered, otherwise no timeline is visible).
# https://doc.qt.io/qt-5/qtwebengine-platform-notes.html#sandboxing-support
```

jonoomph commented:
I do think `launch.py` is an appropriate place for this code. But also, we might try `app.py`, unless that is too-late in the lifecycle for this to be useful. `app.py` might be even better than `launch.py`.

Commit:
Move and Document Dissable Sandbox

- Set Dissable Sandbox in app.py
- Include explanation comment
- Remove Dissable Sandbox from appimage specific script.

jonoomph commented:
LGTM
